apitools/base/py/transfer.py

```diff
@@ -366,10 +366,12 @@ def __ComputeEndByte(self, start, end=None, use_chunks=True):
         end_byte = end
         if use_chunks:
             alternate = start + self.chunksize - 1
-            end_byte = min(end_byte, alternate) if end_byte else alternate
+            end_byte = (min(end_byte, alternate) if end_byte is not None
```

Comment:
probably just make this an `if`/`else`. (line wrapping a ternary operator is hard to read.)